src/rules/media-feature-name-no-unknown/README.md

```diff
@@ -0,0 +1,79 @@
+# media-feature-name-no-unknown
+
+Disallow unknown media feature names.
+
+```css
+@media (min-width: 700px) { }
+/**     ↑
+ * These media feature names */
+```
+
+This rule considers media feature names defined in the CSS Specifications, up to and including Editor's Drafts, to be known.
+
+All vendor-prefixes media feature names are ignored.
+
+Caveat: Media feature names within a range context are currently ignored.
```

Comment:
Please exemplify this or provide a link, as many readers may not know what "a range context" means.